Thuku/React-frontend
# Fetch bucketlists and display them on dashboard

#### What does this PR do?
- Add a feature that fetches bucketlists
#### Description of Task to be completed?
- Fetch bucketlists
#### How should this be manually tested?
- On successful login user should be redirected to the dashboard and a view of their bucketlists
rendered
